docs/ocis/development/beta-testplan.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Beta testplan"
+date: 2022-03-24T00:00:00+00:00
+weight: 37
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/development
+geekdocFilePath: beta-testplan.md
+---
+
+# Beta Testing
+
+This document is supposed to give you some ideas how and what to test on ocis. It's not ment to be an extensive list of all tests to be done, rather it should help you, as beta-tester, to get started and enable you to get creative and create own test-cases. [Derive from these examples, be creative, do unusual and unconventional things, to try to break things](https://twitter.com/sempf/status/514473420277694465).
+
+One option to create new test-cases and to stress the system is to examine what the [API acceptance-tests](https://owncloud.dev/ocis/development/testing/#testing-with-test-suite-natively-installed) or the [web-UI](#web) does, [examine the requests](#decode-https-traffic-with-wireshark) and do something a bit different with curl. This is also a good way to find out how APIs work that are not already fully documented.
+
+Some cases have suggested setup steps, but feel free to use other setups. This can include:
+- different deployment methods (e.g. running single binary, docker-container, docker-compose setup, [individual services in own docker containers](https://owncloud.dev/ocis/deployment/ocis_individual_services/))
+- different identity managers (e.g. [different external LDAP](https://owncloud.dev/ocis/deployment/ocis_ldap/), internal IDM)
+- different reverse proxies (e.g. [traefik](https://owncloud.dev/ocis/deployment/ocis_traefik/))
+- different IdPs **TODO documentation link**
+- different storage systems ([cephfs](https://owncloud.dev/ocis/storage-backends/cephfs/), decompose fs, [decompose fs on NFS](https://owncloud.dev/ocis/storage-backends/dcfsnfs/), [EOS](https://owncloud.dev/ocis/storage-backends/eos/), [S3](https://owncloud.dev/ocis/deployment/ocis_s3/) ) **are that the things we want to support?**
```

Comment:
Currently Supported: decomposedFS on local POSIX, decomposedFS on NFS, S3NG

I would not mention EOS and CephFS because that is not in the scope of the ocis GA